inc/breadcrumb-links.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/**
+ * Breadcrumb Links
+ */
+
+namespace WMF\Breadcrumb_Links;
+
+/**
+ * Kick it off.
+ */
+function init() {
+	add_action( 'add_meta_boxes', __NAMESPACE__ . '\\add_breadcrumb_link_meta_box' );
+	add_action( 'save_post', __NAMESPACE__ . '\\save_breadcrumb_link_custom_fields' );
+}
+
+
+/**
+ * Add the breadcrumb_link meta box.
+ */
+function add_breadcrumb_link_meta_box() {
+	add_meta_box(
```

Comment:
More important than internationalization: Does this have to use a legacy metabox? Why couldn't this use a JS-side control to render the form? I'm worried that adding a legacy metabox these days counts as tech debt. Curious for @joeleenk or @goldenapples's thoughts

Reply:
@kadamwhite regarding to your question above, I decided for a more traditional approach implementing a custom metabox instead of a JS block editor plugin because it's relatively straightforward to be implemented, metaboxes are still fully supported and widely used. I believe that the risk of using this approach is generally low, considering WordPress backwards compatibility. Anyway, I can definitely change it, no problems.

Reply:
Hmm, I can see both sides of this. It's true that metaboxes are still supported and used, but I would generally expect new development for the block editor to be JS-based. If I were going looking for it, that's where I'd expect to find it. It does feel a bit like tech debt, but I'm not sure if that's real, or just bias toward JS. 🤔

Reply:
Based on conversation in slack, expanding on Joeleen's note : metabox approach is OK for this ticket